packages/flutter/lib/src/scheduler/binding.dart

```diff
@@ -1041,6 +1046,9 @@ mixin SchedulerBinding on BindingBase {
   /// statements printed during a frame from those printed between frames (e.g.
   /// in response to events or timers).
   void handleBeginFrame(Duration? rawTimeStamp) {
+    if (!isReadyForDoFrame) {
+      return;
+    }
```

Comment:
This seems wrong to me.

Someone might want to prepare a frame even though the window size is not known yet.